docs/rofl/app.mdx

```diff
@@ -36,220 +41,222 @@ chains).
 [Sapphire runtime]: https://github.com/oasisprotocol/docs/blob/main/docs/build/sapphire/README.mdx
 <!-- markdownlint-enable line-length -->
 
-## Repository Structure and Dependencies
+## App Directory and Manifest
 
-:::info
+First we create the basic directory structure for the ROFL app using the Oasis
+CLI. We also need to select the network (in this case `testnet`) and the account
+that will be the initial administrator of the ROFL app (in this case
+`myaccount`):
+
+```shell
+oasis rofl init myapp --network testnet --account myaccount
+```
+
+This will create the `myapp` directory and initialize some boilerplate needed to
+build a TDX container-based ROFL app. The rest of the guide assumes that you are
+executing commands from within this directory.
+
+The command will output a summary of what is being created:
+
+```
+Creating a new ROFL app with default policy...
+Name:     myapp
+Version:  0.1.0
+TEE:      tdx
+Kind:     container
+Deployment 'default':
+  Network:  testnet
+  ParaTime: sapphire
+  Debug:    false
+  Admin:    myaccount
+Created manifest in 'rofl.yaml'.
+Run `oasis rofl create --update-manifest` to register your ROFL app and configure an app ID.
+```
+
+After you complete this guide, the directory structure will look as follows:
+
+```
+myapp
+├── compose.yaml        # Container compose file.
+└── rofl.yaml           # ROFL app manifest.
+```
+
+The manifest contains things like ROFL app metadata, [secrets],
+[requested resources] and can be modified either manually or by using the CLI
+commands.
+
+[secrets]: features.md#secrets
+[requested resources]: resources.md
+
+## Create
+
+Before the ROFL app can be built it needs to be created on chain to assign it a
+unique identifier (the app ID) which can be used by on-chain smart contracts to
+ensure that they are talking to the right app and also gives the app access to a
+decentralized key management system.
+
+Anyone with enough funds can create an app. Currently, this threshold is 10,000
+TEST on Sapphire Testnet and Localnet (funded automatically). ROFL registration
+on Sapphire Mainnet is yet to be enabled.
+
+:::tip
 
-You can find the entire project inside the Oasis SDK repository under
-[`examples/runtime-sdk/rofl-oracle`].
+In order to obtain TEST tokens needed for creating and running your ROFL apps
+use [the faucet] or [ask on Discord]. To make things easier you should
```

Comment:
`the faucet` only gives 10 tokens, so it's not the best suggestion here, where 10,000 tokens are needed :)